jOOQ/src/main/java/org/jooq/conf/Settings.java

```diff
@@ -65,6 +67,22 @@
     protected StatementType statementType = StatementType.PREPARED_STATEMENT;
     @XmlElement(defaultValue = "true")
     protected Boolean executeLogging = true;
+    @XmlElement(defaultValue = "DEBUG")
+    protected Log.Level executeLoggingBindVariables = Log.Level.DEBUG;
```

Comment:
Did you generate these using XJC? Because this file is governed by and derived from https://www.jooq.org/xsd/jooq-runtime-3.10.2.xsd. You should also adapt https://github.com/jOOQ/jOOQ/blob/master/jOOQ/src/main/resources/xsd/jooq-runtime-3.11.0.xsd accordingly.

Reply:
I will rename this to `executeLoggingSqlStringInlined`. This isn't about logging bind variables themselves, but about logging the SQL string with inlined bind variables.